var/spack/repos/builtin/packages/cube-blade/package.py

```diff
@@ -10,14 +10,15 @@ class CubeBlade(AutotoolsPackage):
     """Simple OTF2 trace explorer"""
 
     homepage = "https://www.scalasca.org/software/cube-4.x/download.html"
-    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.5/dist/blade-0.2.tar.gz"
+    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.8/dist/blade-0.5.tar.gz"
+    maintainers("swat-jsc")
 
-    version("0.2", sha256="ab3c5bbca79e2ec599166e75b3c96a8f6a18b3064414fc39e56f78aaae9c165c")
+    version("0.5", sha256="0770cd4c2e1b8f31bdb6dadf39232b709aee869835e12f9e1ac670d0b276a689")
 
-    depends_on("cube@4.5:")
-    depends_on("cubelib@4.5:")
+    depends_on("cube@4.8:+gui")
+    depends_on("cubelib@4.8:")
     depends_on("qt@5.9.1:")
-    depends_on("otf2@2.1.1:")
+    depends_on("otf2@3.0:")
```

Comment:
According to the docs (https://www.scalasca.org/scalasca/software/cube-4.x/download.html)  `scorep@7:` is also required for 0.5 but I see it explicitly called out for `ScorePion` and `Blade`.  Do you know if the other packages don't actually require it?

Reply:
blade requires only  otf2 and cubegui ....  but otf2 is also shipped with scorep (tarball), hence on web site we mention   scorep  OR otf2...

as spack provides own otf2 package, i guess it is sufficient to put dependency only otf2

scorepion requires the  scorep , and not otf2